### DISCUSSION TITLE
Telemetry Sender and stack file path redaction

### DISCUSSION THREAD

**Discussion by mikeburgh** [microsoft/vscode-discussions#2429] [Extension Development QnA]
When using the vscode.TelemetrySender to capture and trace errors, the stack trace file paths are all: 
REDACTED: user-file-path

Which I understand for not leaking user names and other PII, but is there away to get the filename ? 

Without the filename, the stack is hard to determine which file the error was in (especially when the files are minimized and hence method names are no longer meaninful.

**Comment by robconery** [comment]
I believe the file name is redacted for privacy reasons - so no, there's no way to do this. @isidorn weigh in if I'm wrong.

**Reply by isidorn** [reply]
Correct. This is by design. @lramos15 might know if there is a workaround

**Reply by mikeburgh** [reply]
That makes sense, but if just the file name could be provided that would not have any impact on privacy and make the trace a lot more useful.

eg something like
/redacted/extension.js
